server/models/Notifications.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const notificationsSchema = new mongoose.Schema({
+  notifications: [
+    {
+      isNewNotification: {
+        type: Boolean,
+        default: true,
+      },
+      title: String,
+      body: String,
+      date: Date,
+      from: {
+        firstName: String,
+        lastName: String,
+      },
```

Comment:
Can you make this a ref to the profile? Would make it easier to handle dynamically

Reply:
Yeah that's a good point, I can definitely change that.